client/app/pages/queries/QuerySource.jsx

```diff
@@ -1,17 +1,16 @@
-import { isEmpty, find, map, extend, includes } from "lodash";
-import React, { useState, useRef, useEffect, useCallback } from "react";
+import { extend, find, includes, isEmpty, map } from "lodash";
```

Comment:
Just noticed that imported symbols are sorted alphabetically. Is that Prettier who did it?

Reply:
No, I think I copied the file from the fork like that 🤔

Reply:
Im asking because that's not the only place where I noticed that (like the one in my comment above). Interesting 🤔